# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/credentialmanagement/tests/browser/browser.toml
# Commit: 459739221a88
# Full Hash: 459739221a884b2b1c2adab42fd19fafe279bd20
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing build bustages
#   
#   This reverts commit ab681e36aeafc21ba62bb4680bdaa6206f02546d.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r fc9986230203 -r 459739221a88 toolkit/components/credentialmanagement/tests/browser/browser.toml
--- a/toolkit/components/credentialmanagement/tests/browser/browser.toml	Thu Jun 05 00:15:19 2025 +0300
+++ b/toolkit/components/credentialmanagement/tests/browser/browser.toml	Thu Jun 05 00:19:41 2025 +0300
@@ -2,6 +2,8 @@
 head = "head.js"
 prefs = [
   "dom.security.credentialmanagement.identity.enabled=true",
+  "dom.security.credentialmanagement.identity.heavyweight.enabled=true",
+  "dom.security.credentialmanagement.identity.lightweight.enabled=true",
   "dom.security.credentialmanagement.identity.ignore_well_known=true",
   "privacy.antitracking.enableWebcompat=false",
 ] # disables opener heuristic
@@ -11,4 +13,6 @@
 
 ["browser_account_dialog.js"]
 
+["browser_credential_chooser.js"]
+
 ["browser_provider_dialog.js"]